All Core Devs Meetings/Meeting 87.md

```diff
@@ -0,0 +1,406 @@
+### Moderator: Hudson Jameson
+### Notes: Jim Bennett
+
+---
+
+# Summary
+
+## EIP Status
+
+EIP | Status
+---|---
+EIP-2537, EIP-2315 | `Accepted` Targetting Berlin
+EIP-1057, EIP-1803, EIP-1559 | `Accepted`
+EIP-2464 | `Last Call`
+EIP-1380, EIP-2046,  EIP-2456, EIP-2315, EIP-2541, EIP-2539, EIP-1985, EIP-1702, EIP-1380, EIP-663 | `Eligible for Inclusion`
+EIP-1702 | `Eligible for Inclusion` Pending Champion. Not scheduled for Berlin
+EIP-663, EIP-2348, UNGAS | Depends on EIP-1702
+EIP-1962,  EIP 2456, EIP 2515 | Discussed under `EFI`. Discussion to be continued in EthMagician thread |
+EIP-2515 | Discussed under `EFI`. Decision required around needing a Hard Fork
+
+*Note—Removed EIPs included in Muir Glacier.*
+
+## Decisions Made
+
+Decision Item | Description
+--|--
+**87.1**   | Ephemeral testnet name YOLO (you only live once) to test these Berlin EIPs
+**87.1.1** | 2315, 2537
+
+## Actions Needed
+
+Action Item | Description
+---|---
+**86.1** | Decision required on looking at the contract code for Simple Subroutines.
+**86.2** | James to reach out to Alexi for merkalization in Simple Subroutines.
+**86.3** | ProgPoW compromise proposal to be decided on (testnet then pocket).
+**86.4** | Update EIP bot with Withdrawn
+**86.5** | Move EIP-2583 to Withdrawn
+
+
+---
+
+# Agenda
+
+<!-- MDTOC maxdepth:6 firsth1:1 numbering:0 flatten:0 bullets:1 updateOnSave:1 -->
+
+- [1. Berlin EIPs - Integration Updates](#1-berlin-eips-integration-updates)   
+- [2. Eligible for Inclusion (EFI) EIP Review](#2-eligible-for-inclusion-efi-eip-review)   
+- [3. Berlin Timing](#3-berlin-timing)   
+- [4. EIP-1057: ProgPoW, a Programmatic Proof-of-Work.](#4-eip-1057-progpow-a-programmatic-proof-of-work)   
+- [5. EIP-2565: Repricing of the EIP-198 ModExp precompile. Specifically the open-ethereum benchmarking results per this comment.](#5-eip-2565-repricing-of-the-eip-198-modexp-precompile-specifically-the-open-ethereum-benchmarking-results-per-this-comment)   
+- [6. PR to change "hard fork" to "network upgrade" in EIP-1 and common vernacular](#6-pr-to-change-hard-fork-to-network-upgrade-in-eip-1-and-common-vernacular)     
+- [7. State tree format conversion with the tree overlay method](#7-state-tree-format-conversion-with-the-tree-overlay-method)   
+- [8. EIP-2315: “Simple Subroutines for the EVM” analysis](#8-eip-2315-“simple-subroutines-for-the-evm”-analysis)   
+- [9. EIP-1559 implementers' call update](#9-eip-1559-implementers-call-update)   
+- [10. EIP-2583: Penalty for account trie misses](#10-eip-2583-penalty-for-account-trie-misses)    
+- [11. Testing updates](#11-testing-updates)   
+- [12. Review previous decisions made and action items (if notes available)](#12-review-previous-decisions-made-and-action-items-if-notes-available)   
+
+
+---
+
+
+# 1. Berlin EIPs - Integration Updates
+
+Video | [5:36](https://youtu.be/MOZ7_0Tb95M?t=336)
+-|-
+
+The EIP for changing the difficulty bomb is possible for Berlin, but still needs a lot of work.
```

Comment:
This EIP is no longer going into Berlin and the author has requested it to be withdrawn.